src/bootloader/bootloader.c

```diff
@@ -929,7 +959,7 @@ void bootloader_jump(void)
 #ifdef BOOTLOADER_DEVDEVICE
 #if PLATFORM_BITBOXBASE == 1
         // We don't have touch on base dev bootloader yet
-        (void)_devdevice_enter;
+        _base_wait_fwupdate();
```

Comment:
Could you remove all special handling in `bootloader_jump()`. Since this PR will add the missing functionality (communication to the bootloader). I think it makes sense to revert to the bitbox02 behaviour.

We should stay in the bootloader until we are instructed to boot via the `OP_REBOOT` command. If it is a dev bootloader we should print the same text as on BB02 and wait for user to either boot or to enter bootloader.

You can probably just check out this file from the `upstream/master` branch.

Reply:
Semi-fixed - Some changes are still necessary for the USART to be initialized instead of the USB stack.